docs/cooperation/cooperation.yaml

```diff
@@ -0,0 +1,304 @@
+      requestBody:
+        required: true
+        description: Provide required data to update status of cooperation.
+        content:
+          application/json:
+            schema:
+              $ref: '#/definitions/cooperation'
+            example:
+              offerId: 64ebc6fbd2f34037d0aba782
+              tutorId: 6755bc080a75adf9223df210
+              studentId: 6655bc080a75adf9223df108
+              price: 150
+              status: closed
+      responses:
+        204:
+          description: No Content
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 400
+                code: INVALID_ID
+                message: ID is invalid.
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 401
+                code: UNAUTHORIZED
+                message: The requested URL requires user authorization.
+        403:
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 403
+                code: FORBIDDEN
+                message: You do not have permission to perform this action.
+        404:
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 404
+                code: COOPERATION_NOT_FOUND
+                message: Cooperation with the specified id was not found.
+    delete:
+      security:
+        - bearerAuth: [ ]
+      tags:
+        - Cooperations
+      summary: Find and delete cooperation by ID
+      description: Finds and deletes a cooperation with the specified ID.
+      produces:
+        - application/json
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: ID of the cooperation that needs to be deleted
+          type: string
+      responses:
+        204:
+          description: No Content
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 400
+                code: INVALID_ID
+                message: ID is invalid.
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/Error'
+              example:
+                status: 401
+                code: UNAUTHORIZED
+                message: The requested URL requires user authorization.
+        403:
```

Comment:
it may have sense but need to think who will be able to delete it? admins only? or all users?